main/guides/js-programming/far.md

```diff
@@ -0,0 +1,95 @@
+use [`E()`](./eventual-send.html#remote-object-communication-with-e). For example, getting
+an `Issuer`'s `brand` would look like `E(issuer).getBrand()`.
+
+Objects intended to be used in other vats are called *remotables*. Remote messages sent to
+remotables must only contain *passable* arguments and return *passable* results. 
+Passables includes all things that can be passed as arguments in messages. 
+
+There are three kinds of passables:
+   * Remotables, objects with methods that can be called remotely using `E()`.
+   * Pass-by-copy data, such as numbers or hardened records
+   * Promises for passables.
+
+In particular, note that every object returned from a smart contract, such a `publicFacet` or 
+`creatorFacet`, must be passable. All objects used in your contract's external API must
+be passable.
+
+### Rules for creating remotables
+- All property values must be functions. 
+  - They cannot be accessors.
+  - Note: If you wish to send data, send pass-by-copy data such as pass-by-copy records, 
+    pass-by-copy arrays, strings, numbers, etc. or some other passable.
+- You must wrap the object with `Far()`.
+
+**Note**: ERTP objects, such as `Purses`, are automatically created as `Remotable`, as are
+`UserSeats` and `ZCFSeats`. 
+
+### Using remotables
+- Call a remotable's method by first wrapping the remotable object with `E`, such as `E(issuer).getBrand();`
+- Handle the resulting promise. Calling `E()` always results in a `Promise`.
+
+## Using `Far()`
+
+`Far(farName, object-with-methods)`
+- `farName` `{ String }`
+- `object-with-methods` ` { Object }` `[remotable={}]`
+-  Returns: A `Remotable` object.
+
+The `farName` parameter gives the `Remotable` an *interface name* for debugging purposes, which only shows
+up when logged through `console.log`. 
+
+The `object-with-methods` parameter includes a record with definitions of all the object's 
+property functions. See the example code below.
+
+`Far()` function marks an object as remotable.  `Far()` also:
+- Hardens the object.
+  - Both `harden()` and `Far()` function harden the object. 
+  - Only hardened objects are passable.
+- Checks for the property and value requirements above. 
+  If they are not met, it throws an error.
+- Records the object's interface name. 
+
+You should call `Far()` on an object if it both:
+- Will be sent out of its native vat.
+  - If it might ever appear as the `foo` in [`E(foo).method(args)`](./eventual-send.md),  
+    you should run `Far()` on it after creating it.
+- Has methods called on it, as opposed to just effectively storing data.
+
+As defined, only passables can be either passed as arguments or returned as results,
+and they must be hardened. If the passable is a remotable, it must be hardened with `Far()`.
+Otherwise, it must be hardened with [`harden()`](./ses/ses-guide.md#harden).
+
+There's no harm in using `Far()` on an object, even if it never leaves its vat. An error
```

Comment:
```suggestion
An error
```
Below you correctly explain a harm so we shouldn't start out saying there is none, and then contradict ourselves a moment later.